schema/proto3/gateway.proto

```diff
@@ -50,8 +51,17 @@ message GatewayConfiguration {
         uint32 port_inband_operation = 1;
     }
 
+    message arion {
+            string vpc_id = 1;
+            uint32 vni = 2;
+            string subnet_id = 3;
+            // port for in-band (same NIC channel) operation
+            uint32 port_inband_operation = 4;
```

Comment:
Is this a port id or?

Reply:
This is the port number for the in-band operation. Say, `port_inband_operation=1234`, then when ACA receives this Gateway State, it creates a UDP server at this port(1234) to receive the OAM packet sent by an Arion Wing.

Reply:
@zzxgzgz thank you!

Reply:
@zzxgzgz based on the new design, the OAM packet is sent by a local OAM agent, not the Arion Wing, right?